src/cargo/core/features.rs

```diff
@@ -819,7 +818,6 @@ impl CliUnstable {
             "terminal-width" => self.terminal_width = Some(parse_usize_opt(v)?),
             "namespaced-features" => self.namespaced_features = parse_empty(k, v)?,
             "weak-dep-features" => self.weak_dep_features = parse_empty(k, v)?,
-            "extra-link-arg" => self.extra_link_arg = parse_empty(k, v)?,
```

Comment:
Instead of removing this, can you move it down to the bottom, and include a stabilization message like the other flags?  The version should be 1.56.